dts/arm/rpi_pico/rp2040.dtsi

```diff
@@ -81,6 +81,16 @@
 			label = "UART_1";
 			status = "disabled";
 		};
+
+		usbd: usbd@50100000 {
+			compatible = "raspberrypi,pico-usbd";
+			reg = <0x50100000 0x10000>;
+			interrupts = <5 3>;
```

Comment:
Maybe rename this to `RPI_PICO_DEFAULT_IRQ_PRIORITY` it is set to 3 by default. Or is there a reason this should be independent?

Reply:
Good catch, fixed.